src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -269,7 +269,7 @@ export default class FilterOptionsComponent extends Component {
     this.showMoreState = this.config.showMore;
 
     if (this.config.storeOnChange) {
-      this.apply();
+      this.apply(true);
```

Comment:
Same thing about wanting this behavior if `FilterOptions` is not being used for a `Facet`.